### PR TITLE
pytorch.proto and respective changes

### PR DESCRIPTION
**Pull request by igorvalko**
# TL;DR
proto files and generated ones for pytorch Flyte plugin

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [X] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [X] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
That's the part of pytorch Flyte plugin implementation.
Related PRs:
https://github.com/lyft/flytekit/pull/112
https://github.com/lyft/flyteplugins/pull/85

## Tracking Issue
N/A

## Follow-up issue
N/A

### CODE REVIEW

File: protos/flyteidl/plugins/pytorch.proto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/plugins";
+
+message PyTorchOperatorTask {
```
Comment (EngHabu) [inline protos/flyteidl/plugins/pytorch.proto:7]: Can you add some code comments here...

Comment (igorvalko) [reply protos/flyteidl/plugins/pytorch.proto:8]: Sure, done.

Comment (kumare3) [reply protos/flyteidl/plugins/pytorch.proto:8]: I think we should call the plugin type, DistributedPytorchTraining?

Comment (kumare3) [reply protos/flyteidl/plugins/pytorch.proto:8]: Also the comment then should be, this plugin enables distributed training for pytorch using "link to the Kubeflow operator" - What do you think @igorvalko ?

Comment (igorvalko) [reply protos/flyteidl/plugins/pytorch.proto]: done

**Pull request merged**